Framework/DataHandling/test/SaveGSSTest.h

```diff
@@ -115,6 +115,28 @@ class SaveGSSTest : public CxxTest::TestSuite {
     AnalysisDataService::Instance().remove(wsName);
   }
 
+  void test_unwritablePath() {
+    if (Poco::File("/usr").exists()) {
```

Comment:
Unless I'm mistaken, this means this test will always be skipped on windows. Is this behaviour deliberate?

Reply:
You are correct. I wasn't sure of a location on windows that both exists and isn't user writable. If you have a location, I can modify the test. I don't know if it is a big deal since it is run everywhere else.

Reply:
Good point I can't say I know of an equivalent on windows myself either. I'm happy to approve this now I understand the reasoning.